deploy/iso/minikube-iso/configs/minikube_aarch64_defconfig

```diff
@@ -1,116 +1,4263 @@
-# Architecture
+#
+# Automatically generated file; DO NOT EDIT.
+# Buildroot 2025.02-dirty Configuration
+#
+BR2_HAVE_DOT_CONFIG=y
+BR2_EXTERNAL_NAMES="MINIKUBE"
+BR2_EXTERNAL_MINIKUBE_PATH="/home/jtang/workspace/minikube/deploy/iso/minikube-iso"
+BR2_EXTERNAL_MINIKUBE_VERSION="-gdc4979ad6"
+BR2_HOST_GCC_AT_LEAST_4_9=y
+BR2_HOST_GCC_AT_LEAST_5=y
+BR2_HOST_GCC_AT_LEAST_6=y
+BR2_HOST_GCC_AT_LEAST_7=y
+BR2_HOST_GCC_AT_LEAST_8=y
+BR2_HOST_GCC_AT_LEAST_9=y
+BR2_HOST_GCC_AT_LEAST_10=y
+BR2_HOST_GCC_AT_LEAST_11=y
+BR2_NEEDS_HOST_UTF8_LOCALE=y
+
+#
+# Target options
+#
+BR2_ARCH_IS_64=y
+BR2_USE_MMU=y
+# BR2_arcle is not set
```

Comment:
Why do we have the defaults in this file? I think that buildroot removes the defaults to minimize the configs. Can you explain how the file was generated?